File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //ask user height integer
    int height;
    do
    {
        height = get_int("What height do you want the stairs to be? ");
    }
    while (height < 1 || height > 8);
}

File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height < 1 or height > 8:
        height = get_int("Height: ")
    else:
        break


for i in range (1, height + 1):
    print((' ') * (int(height) - i) + ('#') * (i))
    i += 1
